components/menu/native-menu/native-menu.scss.js

```diff
@@ -0,0 +1,174 @@
+import { scss } from "../../../deps.ts";
+
+export default scss`
+.c-native-menu {
+  height: 100%;
+  width: 100%;
+  position: relative;
+
+  --error-red: rgba(238, 113, 113, 1);
+  --success-green: rgba(107, 190, 86, 1);
+  // TODO: rm this when we either include it in the design system or have it as a component prop
+  --truffle-gradient: linear-gradient(281.86deg, #71DBDB 2.63%, #ADACDD 50.48%, #FF9DC6 94.5%);
+  $clip-path-transition: .5s;
+
+  &.position-top-left, &.position-bottom-left {
+    > .menu {
+      > .inner {
+        > .bottom {
+          > .c-tab-bar {
+            flex-direction: row;
+            order: 2;
+          }
+          > .c-extension-icon {
+            order: 1;
+          }
+        }
+      }
+    }
+  }
+
+
+
+  &.is-collapsed {
+    > .menu {
+      /* TODO: consolidate the button styles */
```

Comment:
Is this something to be done before merging the PR?

Reply:
nah, I just need to cleanup the comment. the button styles diverged pretty significantly w/ the final mocks